flink-filesystems/flink-swift-fs-hadoop/src/main/resources/META-INF/NOTICE

```diff
@@ -50,11 +50,6 @@ Written by Brian Goetz and Tim Peierls with assistance from members of JCP JSR-1
 and released to the public domain, as explained by the Creative Commons public domain license.
 https://creativecommons.org/licenses/publicdomain/
 
-This project bundles "org.tukaani:xz:1.5".
```

Comment:
the pom still defines a relocation pattern.

I'm wondering how long this entry has been incorrect; even in 1.10 it doesn't show up in the shade plugin output...

Reply:
I'll remove it from the pom